### ISSUE TITLE
[Question] Block negative stock value

### ISSUE THREAD

**Issue by liviubbo** [YetiForceCompany/YetiForceCRM#9494]
Hello ! 

    I want to block Goods Delivery Notes for products that are ZERO in inventory.
   For example, if a product has value zero in quantity, i want to block the IGDN for that product, until it goes back to stock.

**Comment by liviubbo** [comment]
I have created a workaround, if someone is interested in this.
So, what i wanted originaly is, to block Goods Delivery Notes for products that have zero quantity in inventory. I wanted this for a variety of reasons but mainly because i had some products that entered phisicaly in stock without Goods Received notes. Why that happened, right now is unclear, but i wanted to block the posibility to exit stock and have negative quantity for products.

I have set the products module to private. I have created an workflow that changes the owner of the products that have zero or negative value to a user created by me specialy for this. 
I have added some rules that gives anyone that works with products permission to create/edit records by anyone except the user specialy created for this. 
What i acomplished is that products with zero quantity or minus (logicaly this should not happen, but it happens) cannot be seen/user by anyone but the administrator and user special created for this.
